src/applications/vaos/new-appointment/redux/helpers/formSubmitTransformers.js

```diff
@@ -107,9 +110,12 @@ export function transformFormToVARequest(state) {
     ...getRequestedDates(data),
     // The bad camel casing here is intentional, to match downstream
     // system
-    bestTimetoCall: Object.entries(data.bestTimeToCall)
-      .filter(item => item[1])
-      .map(item => titleCase(item[0])),
+    bestTimetoCall:
+      flowType === FLOW_TYPES.REQUEST
```

Comment:
I'm not sure you need to make these changes in here, this function should only be called if we're in the request flow.

Reply:
Done